packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -126,26 +126,27 @@ class DeactivateAccountReason extends React.Component {
     validateFields = values => {
         const error = {};
         const selected_reason_count = selectedReasons(values).length;
+
         if (!selected_reason_count) {
             error.empty_reason = localize('Please select at least one reason');
         }
         if ((values.otherTradingPlatforms + values.doToImprove).length > 0 || selected_reason_count) {
             const max_characters = character_limit_no;
             const final_value = preparingReason(values);
 
-            const remaining_characters = max_characters - final_value.length;
-            if (remaining_characters >= 0) {
-                this.setState({ remaining_characters });
-            } else {
-                this.setState({ remaining_characters: 0 });
-            }
+            const text_inputs_length = values.otherTradingPlatforms.length + values.doToImprove.length;
+            let remaining_characters = max_characters - text_inputs_length;
```

Comment:
Since `max_characters` is `character_limit_no`, let's make it: 

```suggestion
            let remaining_characters = character_limit_no - text_inputs_length;
```

Reply:
pushed the change